### PR TITLE
doc: Add a bullet for bond managing with NNCP

### PR DESCRIPTION
**Pull request by qinqon**
The repository contains a pair of yamls to create and delete bond but is
missing how to do it on the README. This change add a bullet with the
procedure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,10 +48,14 @@ export base64_script_content=$(cat init-interfaces.sh|base64 -w 0) && envsubst <
 - Follow the guide to install a bare-metal cluster in the [OpenShift production documentation.
 ](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/installing/installing-on-bare-metal)
 
+### 4. Apply the kubernetes-nmstate NNCP Bond configuration
+- Create bond `oc apply -f add-slb-nncp.yaml`
+- Delete the bond `oc delete nncp --all && oc apply -f del-slb-nncp.yaml`
```
Comment (phoracek) [inline README.md:53]: Should we list this as a standalone note? Having it here may risk that users would copy copy copy everything without realizing they are removing what they just created.

Comment (qinqon) [reply README.md:53]: Ahh you mean Put the delete as a note right ? ok, make sense

**Review by phoracek — APPROVED** [review]

**Comment by RamLavi** [comment]
too late but still :+1:

**Pull request merged**